pkgs/development/lua-modules/overrides.nix

```diff
@@ -174,18 +176,14 @@ in
 
       # TODO: remove with new nvim-web-devicons release
       # Disabled devicons test because we have old version as dep and fzf-lua checks for a new icon
```

Comment:
Is this still valid?

Reply:
yeah, they haven't done a release since may 2024. :S